cmd/beacon-chain/flags/base.go

```diff
@@ -35,13 +35,13 @@ var (
 		Value: "127.0.0.1",
 	}
 	// RPCPort defines a beacon node RPC port to open.
-	RPCPort = &cli.IntFlag{
+	RPCPort = &cli.Uint64Flag{
```

Comment:
please review what the integer was originally for. 
in the example, a port number is a 16-bit unsigned integer, not a 64-bit unsigned integer, and should be used as such. you'll also need to take a look at Raul's PR and what he did for the flags because the change to the flag may affect the flag value and how it's used so it may need to have certain casting for it)